7.analyze-coverage/coverage-analysis.py

```diff
@@ -3,36 +3,29 @@
 Interpret Compression
 7.analyze-coverage/coverage-analysis.py
 
-Track and visualize gene set activity across compressed features
+Track gene set coverage across compressed features
 
 Usage:
 
     python coverage-analysis.py
 
 Output:
-Several geneset enrichment scores and many figures describing enrichment across features
+Several geneset coverage scores summarizing enrichment across features
 """
 
 import os
 import subprocess
 
-datasets = ['GTEX', 'TCGA', 'TARGET']
-metaedges = ['GpXCELL', 'GpH', 'GpC1', 'GpC2CPG', 'GpC2PREACTOME', 'GpC3MIR', 'GpC3TFT',
-             'GpC4CGN', 'GpC4CM', 'GpC5BP', 'GpC5CC', 'GpC5MF', 'GpC6', 'GpC7']
-
-for dataset in datasets:
-    for metaedge in metaedges:
-        metaedge_lower = metaedge.lower()
-        output_dir = os.path.join('results', dataset.lower(), metaedge_lower)
-
-        coverage_command = ['python', 'get-coverage.py',
-                            '--dataset', dataset,
-                            '--geneset', metaedge_lower]
-        visualize_command = ['Rscript', '--vanilla', 'visualize_coverage.R',
-                             '--dataset', dataset,
-                             '--geneset', metaedge_lower,
-                             '--output_dir', output_dir]
-        print(coverage_command)
-        subprocess.call(coverage_command)
-        print(visualize_command)
-        subprocess.call(visualize_command)
+coverage_pairs = [('TCGA', 'GpC5BP'),
+                  ('TCGA', 'GpC4CM'),
+                  ('GTEX', 'GpXCELL')]
+
+for dataset, metaedge in coverage_pairs:
+    metaedge_lower = metaedge.lower()
+    output_dir = os.path.join('results', dataset.lower(), metaedge_lower)
+
+    coverage_command = ['python', 'get-coverage.py',
+                        '--dataset', dataset,
+                        '--geneset', metaedge_lower]
+    print(coverage_command)
+    subprocess.call(coverage_command)
```

Comment:
I haven't seen this notation before.  Is this just saying to run all the functions in the "get-coverage.py" file?  Seems a bit strange to me because you're not calling a specific function but multiple ones in a given file.

Reply:
Yeah, I am using `coverage-analysis.py` in a similar way bash scripts are used to call a whole bunch of additional scripts

Reply:
so `coverage-analysis.py` makes three calls to `get-coverage.py`